Packs/AzureActiveDirectory/ReleaseNotes/1_2_2.md

```diff
@@ -0,0 +1,3 @@
+
+#### Incident Fields
+- **Azure Active Directory Identity and Access Detected Date Time**
```

Comment:
@yaakovi @reutshal  This is the correct format? not line by line anymore?

Reply:
yes